### PR TITLE
chore: bump the required node version

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Bumps the required Node version to v18 as v16 is no longer maintained. This change also makes the CI and local environments consistent as we have already been running our CI jobs on Node 18.

### CODE REVIEW

**Comment by raisedadead** [comment]
We might want to jump straight to v20 correct @ojeytonwilliams?

**Comment by ojeytonwilliams** [comment]
Yeah, it's becoming LTS on the 24th, so I think we can go to 20 now.

For the CI, it's a little awkward. In an ideal world we'd test against both 18 and 20 until we're using 20 in production, but we're already queuing up a ton of actions as it is. Maybe we should just move to 20 in CI, too. What do you reckon?

**Comment by raisedadead** [comment]
Yes - I agree

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "license": "BSD-3-Clause",
   "private": true,
   "engines": {
-    "node": ">=16",
+    "node": ">=18",
```
Comment (ojeytonwilliams) [inline package.json:8]:
```suggestion
    "node": ">=20",
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Alrighty, then let's go with 20 here and I'll create a quick PR for CI.

**Comment by ojeytonwilliams** [comment]
Absolutely. Feel free to close my PR in favour of this one, if that works for you.

**Comment by ojeytonwilliams** [comment]
Thanks, @huyenltnguyen!

**Pull request closed without merging**